### DISCUSSION TITLE
Azure support

### DISCUSSION THREAD

**Discussion by jfmatth** [ehmatthes/django-simple-deploy#246] [General]
I heard you on my Python podcast and you mentioned Azure support was coming or in Beta, but I don't see it.

J

**Comment by ehmatthes** [comment]
Hi J. I built provisional support for Azure into an earlier version of the project, but it was too complex to continue supporting in the pre-1.0 phase of this project. You can read more about that decision in the Rationale section of #83.

If you have any responses to what you see there, I would be curious to hear your perspective.

**Reply by jfmatth** [reply]
Thank you @ehmatthes  for the quick response.  Love the project, and Ill help where I can
